### PR TITLE
fix: bug 1958 missing entity showing id not name

### PR DESCRIPTION
**Pull request by LarsLiden**

### CODE REVIEW

File: src/FilledEntity.ts [hunk 1]
```diff
@@ -38,11 +38,26 @@ export const getEntityDisplayValueMap = (filledEntityMap: FilledEntityMap): Map<
...
     }
 
+    // Entity is missing. Provide readable error message
+    else {
+      // Default to showing key
+      let missingValue = key
+
+      // If key is an entityId, try to get value from name key
+      if (filledEntityMap.map[key].entityId ===  key) {
+        for (let k of Object.keys(filledEntityMap.map)) {
```
Comment (mattmazzola) [inline src/FilledEntity.ts]: Looks like a `find` operation

Comment (LarsLiden) [reply src/FilledEntity.ts]: find operates on arrays.  This is operating on a key / values.   could transform it to an array first, but this seems as good

Comment (mattmazzola) [reply src/FilledEntity.ts]: You did transform it to an array with `Object.keys`  that's the point of the method.

**Pull request merged**